### PR TITLE
add s3:DeleteObject permission while av_delete_infected_files equals true 

### PR DESCRIPTION
**Pull request by liranbenabu1**
IAM role will contain the correct permission to delete file in case the user would like to delete files automatically in case the file found as a infected

### CODE REVIEW

**Comment by liranbenabu1** [comment]
@dynamike @chrisgilmerproj  Hi, someone can take a look into my PR? I saw other people would like to use the code. 
thanks :)

**Comment by chrisgilmerproj** [comment]
@liranbenabu1 - Unfortunately I don't work at this company anymore and cannot review this work for you.

**Comment by rpdelaney** [comment]
Hello! Sorry for the slow-turn around. Is this still needed?

**Comment by lliranbabi** [comment]
Hi , 
It fix some bug you have in the code 
So ... yes I believe it still needed

File: versions.tf [hunk 1]
```diff
@@ -5,3 +5,4 @@ terraform {
     aws = ">= 3.0"
   }
 }
+
```
Comment (rpdelaney) [inline versions.tf:8]:
```suggestion
```

**Review by esacteksab — APPROVED** [review]

**Pull request merged**